src/config/config.ts

```diff
@@ -560,6 +560,27 @@ export class Config implements IConfig {
   }
 }
 
+// when no manifest exists, the default is calculated.  This may throw, so we need to catch it
+const defaultToCached = async (flag: CompletableOptionFlag<any>) => {
+  // Prefer the helpDefaultValue function (returns a friendly string for complex types)
+  if (typeof flag.defaultHelp === 'function') {
```

Comment:
Isn't the goal of this property to be calculated at run time? If so, it needs to be rerun whenever help is generated: https://github.com/oclif/core/blob/main/src/help/command.ts#L290